grease/src/Grease/Options.hs

```diff
@@ -104,6 +112,8 @@ data SimOpts
   , simEnableDWARFPreconditions :: Bool
   -- ^ Enables parsing DWARF to extract initial shape types. This option is
   -- superseded by `simInitialPreconditions` and does not do anything in LLVM mode.
+  , simTypeUnrollingBound :: TypeUnrollingBound
+  -- ^ Bounds the number of times a recursive pointer will be visited when building shapes from DWARF (bounded to 3 recursive pointers by default).
```

Comment:
Maybe:
```suggestion
  -- ^ Bounds the number of times a recursive type will be visited when building shapes from DWARF (bounded to 3 recursive pointers by default).
```
Also, I think there's a way to get `optparse-applicative` to show the default, rather than hardcoding it in the comment.

Reply:
We should remove this from the comment if it is now handled by `showDefault`.